src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIBasic.java

```diff
@@ -1040,6 +1046,9 @@ protected boolean checkOrRequestItems(final boolean useItemDamage, @Nullable fin
         }
         itemsNeeded.clear();
         Collections.addAll(itemsNeeded, items);
+        //emergency workaround so the worker will tell what he needs (ignoring the content of his chest) since he is not able to check it in the issued state
+        final ItemStack itn = itemsCurrentlyNeeded.get(0);
+        requestWithoutSpam(itn.getCount() + " " + itn.getDisplayName()+" Please put it in my Inventory");
```

Comment:
do we have a method that does this @Raycoms 
if not we definitely should have it if we use this at more than one place...